setup.sh

```diff
@@ -202,6 +201,9 @@ az keyvault set-policy --certificate-permissions get --object-id $ASB_POD_MI_ID
 az keyvault set-policy --secret-permissions get --object-id $ASB_POD_MI_ID -n $ASB_KV_NAME
 
 # config traefik
```

Comment:
can we change the name from "manual-test"?

Reply:
👍 i copied the value to a new secret called 'aks-sb-crt'. testing it in my latest deployment.